Mathlib/Topology/Algebra/Group/Basic.lean

```diff
@@ -1895,4 +1896,4 @@ theorem coinduced_continuous {α β : Type*} [t : TopologicalSpace α] [Group β
 
 end GroupTopology
 
-set_option linter.style.longFile 1900
+set_option linter.style.longFile 2100
```

Comment:
Confusion?

Reply:
Nope, apparently the extra line made the `longFile` linter complain. (Maybe there's a `<` in the code that should be a `≤`?)

Reply:
I would at least make it
```suggestion
set_option linter.style.longFile 2000
```
so that it doesn't break next time someone removes a line